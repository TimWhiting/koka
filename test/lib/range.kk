fun main()
  val a = [1, 2, 3].vector
  val x = a[3]
  x.show.println
  ()