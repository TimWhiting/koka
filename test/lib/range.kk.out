uncaught exception: index out of bounds
add default effect for std/core/exn